src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -91,8 +91,7 @@ def load_command_table(self, _):
 
     compute_vm_run_sdk = CliCommandType(
         operations_tmpl='azure.mgmt.compute.operations.virtual_machine_run_commands_operations#VirtualMachineRunCommandsOperations.{}',
-        client_factory=cf_run_commands,
```

Comment:
Please rebase and get rid of the change which have been merged into the public repo

Reply:
Rebased